Src/AutoFixture.xUnit.net.UnitTest/CompositeDataAttributeInsufficientDataTest.cs

```diff
@@ -74,17 +74,7 @@ public IEnumerator<object[]> GetEnumerator()
             yield return CreateTestCase(
                 data: new[]
                 {
-                    new FakeDataAttribute(this.method, this.parameterTypes, new[] {  new object[] { 1, 2    },  new object[] {  3,  4     }, new object[] { 5, 6 } }),

```

Comment:
Correct. – And also the test in the example above fails (I should have mentioned that, I apologize for this).